### PR TITLE
JS codestyle improvements: equality, indentation & etc.

### PR DESCRIPTION
**Pull request by Crasher-ua**
* Changed counterparts operator to equality (eslint rule eqeqeq);
* Reduced complexity by inverting some conditions;
* Fixed indentation near HTML parts;
* Added newlines to few HTML strings;
* Applied correct naming for few one-symbol indentificators (this task should be finished in the next pull request).

### CODE REVIEW

File: src/main/webapp/pipe.js [hunk 1]
```diff
@@ -216,20 +223,18 @@ function pipelineUtils() {
...
             var source;
             var target;
             lastResponse = data;
             equalheight('.pipeline-row .stage');
 
-            Q.each(data.pipelines, function (i, component) {
-                Q.each(component.pipelines, function (j, pipeline) {
-                    index = j;
-                    Q.each(pipeline.stages, function (k, stage) {
+            Q.each(data.pipelines, function (dataPipelineIndex, component) {
```
Comment (Crasher-ua) [inline src/main/webapp/pipe.js]: This task is only started. Idea: rename all one-symbol and two-symbol named variables to something appropriate to debug it faster. If you agree with this idea, I will finish it in the next pull request.

File: src/main/webapp/pipe.js [hunk 2]
```diff
@@ -317,12 +322,14 @@ function isTaskLinkedToConsoleLog(data, task) {
 }
 
 function getPagination(showAvatars, component) {
-    var html = [];
-    if (!showAvatars && component.pagingData != '') {
-        html.push('<div class="pagination">');
-        html.push(component.pagingData);
-        html.push('</div>');
+    if (showAvatars || component.pagingData === '') {
```
Comment (Crasher-ua) [inline src/main/webapp/pipe.js]: Here I have only inverted the condition to reduce nesting. This is common approach, which allows to make code more readable. Probably, will be used in future requests as well

File: src/main/webapp/pipe.js [hunk 3]
```diff
@@ -331,105 +338,119 @@ function getLink(data, link) {
...
+        return undefined;
     }
+
+    var html = ['<div class="infoPanelOuter">'];
+    html.push('<div class="infoPanel"><div class="infoPanelInner">' + task.description.replace(/\r\n/g, '<br/>') + '</div></div>');
+    html.push('</div>');
+    return html.join('');
 }
 
 function generateTestInfo(data, task) {
```
Comment (Crasher-ua) [inline src/main/webapp/pipe.js]: Please note: git shows big difference, but the changes made:
1. Inverted the condition.
2. Made indentation consistent for the lines of block, which previously was in if statement

File: src/main/webapp/pipe.js [hunk 4]
```diff
@@ -331,105 +338,119 @@ function getLink(data, link) {
...
+    return html.join('');
 }
 
 function trimWarningsFromString(label) {
     var offset = label.indexOf('Warnings');
-    return offset == -1 ? label : label.substring(0, offset).trim();
+    return offset === -1 ? label : label.substring(0, offset).trim();
 }
 
 function generatePromotionsInfo(data, task) {
```
Comment (Crasher-ua) [inline src/main/webapp/pipe.js]: This method has same problem with big diff and same changes as in previous comment

File: src/main/webapp/pipe.js [hunk 5]
```diff
@@ -472,7 +492,8 @@ function generateAggregatedChangelog(stageChanges, aggregatedChangesGroupingPatt
             var matches = stageChange.message.match(re) || [unmatchedChangesKey];
 
             Q.unique(matches).forEach(function (match) {
-                (changes[match] || (changes[match] = [])).push(stageChange);
```
Comment (Crasher-ua) [inline src/main/webapp/pipe.js]: And here I have removed inline assignment. Similar to eslint no-cond-assign rule

**Comment by tommysdk** [comment]
@Crasher-ua Thanks for yet another pull request! I will unfortunately not find the time to review this any time soon. Maybe @mrfatstrat @patbos or @pskumar448 could help out and review this.

**Comment by Crasher-ua** [comment]
Hello @tommysdk! I would like to make a bunch of other changes. I can make it commit-by-commit in one pull request or to continue current approach with separate requests. According to the speed of review, can you please advise me the best way?

**Comment by pskumar448** [comment]
Hi @Crasher-ua 
I am going through the changes.

File: src/main/webapp/pipe.js (hunk 1 above)
Comment (pskumar448) [inline src/main/webapp/pipe.js]: @tommysdk 
Confirm is this OK renaming of one-symbol and two-symbol to appropriate name.

Comment (Crasher-ua) [inline src/main/webapp/pipe.js]: Description of this idea can be found in ESLint rule id-length, http://eslint.org/docs/rules/id-length

**Comment by Crasher-ua** [comment]
@tommysdk sorry for inconvenience, but I would like to make more contribution to the project. Current changes are only about codestyle, but following ideas are aimed to upgrade performance and functionality at all. The perfect option would be daily review if it's possible. Can you please advise me the best way of creating requests (for example, bigger amount of commits per one pull request)?

File: src/main/webapp/pipe.js (hunk 1 above)
Comment (tommysdk) [reply src/main/webapp/pipe.js]: Appropriate variable names are preferred

File: src/main/webapp/pipe.js (hunk 2 above)
Comment (tommysdk) [reply src/main/webapp/pipe.js]: Readability is good!

**Comment by tommysdk** [comment]
@Crasher-ua Thank you very much for yet another contribution. I will try to take a closer look at this during next week so that we can merge it. Thanks for your patience!

**Pull request merged**